src/aosm/azext_aosm/_help.py

```diff
@@ -10,60 +10,171 @@
 ] = """
     type: group
     short-summary: Commands to interact with Azure Operator Service Manager (AOSM).
+    long-summary: |
+      Azure Operator Service Manager (AOSM) is a service that enables you to manage and publish Network Function Definitions (NFD) and Network Service Definitions (NSD) to Azure. These commands allow you to build and publish NFDs and NSDs, including supporting infrastructure, from existing Helm charts and ARM templates.
 """
 
 helps[
     "aosm nfd"
 ] = """
     type: group
     short-summary: Manage AOSM publisher Network Function Definitions.
+    long-summary: |
+      A Network Function Definition (NFD) is a collection of Helm charts or ARM templates that define a network function. This command group allows you to build and publish NFDs to Azure.
 """
 
 helps[
     "aosm nfd generate-config"
 ] = """
     type: command
     short-summary: Generate configuration file for building an AOSM publisher Network Function Definition.
+    long-summary: |
+      Generates a configuration file that you can use to build an AOSM Network Function Definition (NFD). The configuration file is a JSONC file that contains the required parameters for building the NFD. You must complete the configuration file with your specific values before building the NFD.
+    parameters:
+      - name: --definition-type
+        type: string
+        short-summary: Type of network function definition to be built.
+        long-summary: |
+          The type of AOSM definition to be built. The generate config file differs depending on type. This can be one of the following values:
+          - vnf: Virtual Network Function
```

Comment:
Maybe say for VNF "on Azure Core" like you have for the other

Reply:
Done.